BEIMA.DB.Schemas/device-schema.txt

```diff
@@ -28,10 +28,8 @@
             "fileName": String
         }
     ],
-    photos: [
-        {
-            "fileUid": String,
-            "fileName": String
-        }
-    ]
+    photos: {
```

Comment:
```suggestion
    photo: {
```
Really small change, but we should be clear that we are only storing one photo per device for now